ansible/inventory/genestack/group_vars/k8s_cluster/k8s-cluster.yml

```diff
@@ -17,7 +17,7 @@ kube_token_dir: "{{ kube_config_dir }}/tokens"
 kube_api_anonymous_auth: true
 
 ## Change this to use another Kubernetes version, e.g. a current beta release
-kube_version: v1.28.6
+kube_version: v1.28.12
```

Comment:
While my comments prove the abilty to upgrade beyond, 1.28.x this change keeps our major version consistent until we're ready to run a major upgrade.

Reply:
Chatted with Kevin and while we are preparing the environment for eventual upgrades, we are pinning the k8's version and only upgrading kubespray.